tests/test_ksuid.py

```diff
@@ -91,7 +91,7 @@ def test_get_payload():
     ksuid = Ksuid()
 
     # Assert
-    assert ksuid.payload == bytes(ksuid)[TIMESTAMP_LENGTH_IN_BYTES:]
+    assert ksuid.payload == bytes(ksuid)[Ksuid.TIMESTAMP_LENGTH_IN_BYTES :]
```

Comment:
```suggestion
    assert ksuid.payload == bytes(ksuid)[Ksuid.TIMESTAMP_LENGTH_IN_BYTES:]
```

Reply:
It was done by black (automatically).

Reply:
We can change our formatter later if we want, but this is what black did.